scripts/tts_dataset_files/cmudict-0.7b_nv22.08

```diff
@@ -77265,7 +77265,8 @@ MERCEDESES(1)  M ER2 S EY1 D IY2 Z
 MERCENARIES  M ER1 S AH0 N EH2 R IY0 Z
 MERCENARY  M ER1 S AH0 N EH2 R IY0
 MERCER  M ER1 S ER0
-MERCHANDISE  M ER1 CH AH0 N D AY2 Z
```

Comment:
Not sure why our cmudict-0.7b did not cover `MERCHANDISE`. But original 0.7b already has the entry, see here: https://raw.githubusercontent.com/Alexir/CMUdict/master/cmudict-0.7b. Could you pls further verify if there are other entries missing in our dict?

Reply:
Did a diff, and it's odd--we're actually missing a random assortment of words. I'll add them back in today for ARPABET. @ekmb -- can you re-build the IPA version once we fix this up?

Reply:
What version did we create our cmudict off of? Was it ADLR? Is there a reason that they were removed?

Reply:
By the way, how about breaking down this PR into two separate ones: one for heteronyms and the other for updating the diff of cmudict?

Reply:
Yes, it was the ADLR version--I'm not entirely sure. Picking some where I'm diffing right now, I'm seeing words like `ANGRINESS`, `ANIMATING`, and `ANIME` that are missing, along with some names.

Reply:
I thought we did the diff between ADLR and cmudict-0.7b already before and consciously made the decision to use ADLR as the base

Reply:
For some reason I remember fewer missing words from the diff we did at the time. Am I mistaken? It seems weird that we would've removed these words and not merged them in as well, since I recall having to do some nontrivial amount of merging then too.

Reply:
Given the number of words that need to be added to merge with the newer 0.7b version of CMUdict, I'd like to make that a new PR.

Reply:
I also helped to merge some entries before, which actually have already existed in the original cmudict. Let's update it in another PR. 

Besides, it would be great if we can add bullet items to the beginning of the cmudict file to make clear version control for any updates of dictionary based on original cmudict and previous revisions. For example,

```
;;; # ========================================================================
;;; #
;;;
;;;  NOTES  ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
;;;  [20220817] new entries added
;;;   - WORDs: a, b,c, d
 
;;;  [20080401] (air)  New dict file format introduced
;;;   - comments (like this section) are allowed
;;;   - file name is major version; vers/rev information is now in the header
;;;
;;;
;;; ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Reply:
An alternative solution is to save any revisions/additions to a separate file, so at least we can have clear organization on what we changed based on original cmudict, for example, we can maintain 3 files,
```
file 1: just a copy of the original cmudict. 
file 2: an addition file including any new entries.
file 3: an revision file including any revisions of original cmudict (file 1).
```

Reply:
@blisc @redoctopus what do you think?

Reply:
I like the first suggestion; I'd rather keep it all as one file to avoid clutter/possible confusion.

Git commit history in addition to header notes should make it clear how the dictionary was edited, and we also have NVIDIA additions/modifications made explicit in two sections at the end of the file.